src/IssueViz.UnitTests/Editor/QuickActions/IssueLocationActionsSourceTests.cs

```diff
@@ -326,7 +353,9 @@ private static IAnalysisIssueVisualization CreateIssueViz(params IAnalysisIssueF
             var testSubject = CreateTestSubject(selectedIssueLocationsTagAggregator.Object, issueLocationsTagAggregator.Object, selectionService.Object);
             var actualActionsSet = testSubject.GetSuggestedActions(null, mockSpan, CancellationToken.None);
 
-            return actualActionsSet.ToList();
+            var hasActions = testSubject.HasSuggestedActionsAsync(null, mockSpan, CancellationToken.None).Result;
```

Comment:
This looks weird: each unit test is now calling two separate public methods on the test subject. At the very least, it makes all of the test names misleading since the names suggest that only `GetSuggestedActions` is being called.

Reply:
Yes but results of both these methods are tied together and it is not expected them to be act independently. Maybe changing the test names would makes sense but separating the tests would only lead to code duplication.

Reply:
Ok, makes sense. Please change the method names for this method and calling tests, and add a comment saying why we're calling two public APIs in the this case.

Reply:
I have made the changes